ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubLiteCompat.java

```diff
@@ -0,0 +1,65 @@
+package com.mozilla.telemetry.transforms;
+
+import com.google.cloud.pubsublite.beam.PubsubLiteIO;
+import com.google.cloud.pubsublite.proto.AttributeValues;
+import com.google.cloud.pubsublite.proto.PubSubMessage;
+import com.google.cloud.pubsublite.proto.SequencedMessage;
+import com.google.protobuf.ByteString;
+import com.mozilla.telemetry.ingestion.core.Constant.Attribute;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.transforms.MapElements;
+import org.apache.beam.sdk.values.TypeDescriptor;
+
+/** Functions for converting messages between standard PubSub and PubSub Lite. */
+public class PubsubLiteCompat {
+
+  /**
+   * Convert message from {@link PubsubLiteIO.read} to {@link PubsubMessage}.
+   * 
+   * <p>For compatiblity with standard PubSub {@link PubsubMessage} extract exactly one value per
+   * attribute, even though Pubsub Lite's {@link PubSubMessage} allows multiple values.
+   */
+  public static PubsubMessage fromPubsubLite(SequencedMessage sequencedMessage) {
+    PubSubMessage message = sequencedMessage.getMessage();
```

Comment:
It's super confusing here that we have both `PubSubMessage` and `PubsubMessage` which differ only by casing. I think we should always refer to the proto type using a fully-qualified name.